src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10012,6 +10016,9 @@ namespace OutputReportTabular {
 				if ( Zone( iZone ).FloorArea > 0 && usezoneFloorArea ) {
 					tableBody( 12, iZone ) = RealToStr( totPlugProcess * Wm2_unitConv / Zone( iZone ).FloorArea, 4 );
 				}
+				// Hybrid Model Multiplier Added by Sang Hoon Lee August 2015
+				HMMultiplier = Zone( iZone ).ZoneVolCapMultpSensHMAverage;
```

Comment:
For zones that are not hybrid, this should report the "standard" capacitance multiplier Zone( ZoneNum ).ZoneVolCapMultpSens  Also, it would be useful to indicate which zones are hybrid, but this report already has many columns.  Maybe the "Conditioned" column could be Yes/No/Hybrid?
Also, any changes to output tables need to be described in /src/Transition/OutputRulesFiles/OutputChanges8-6-0-to-8-7-0.md  See example [here](https://github.com/NREL/EnergyPlus/blob/develop/src/Transition/OutputRulesFiles/OutputChanges8-5-0-to-8-6-0.md)